lib/debug/thread_client.rb

```diff
@@ -7,6 +7,17 @@
 require_relative 'color'
 
 module DEBUGGER__
+  module SkipPathHelper
+    def skip_path?(path)
+      (skip_path = CONFIG[:skip_path]) && skip_path.any?{|skip_path| path.match?(skip_path)}
```

Comment:
```suggestion
      (skip_path = CONFIG[:skip_path]) && skip_path.any?{|path| path.match?(path)}
```

use different var name (sorry my previous suggestion missed it).